### DISCUSSION TITLE
Is the parts offset information correct?

### DISCUSSION THREAD

**Discussion by lofti198** [Tyrrrz/YoutubeExplode#771] [Q&A]
Noticing that sometimes offsets are not corresponding to real duration of particular parts stadying one by one together. It is critical for me, since I try to use them to split subtitle to logical blocks like sentences or paragraphs, using this offsets. Havent you notice the same?

**Comment by Tyrrrz** [comment]
Can you show an example of what you mean?

**Reply by lofti198** [reply]
Thank you for fast responce! For instance:
https://youtu.be/85qHi4jIUIk?t=38

There is 1 second difference between offsets of words "this" and "is" while if you listen to the phrase (follow the link above) - you won't here any big pause. Or maybe I somehow misunderstand the meaning of offset Property

![image](https://github.com/Tyrrrz/YoutubeExplode/assets/42960488/d199dfb4-e1b4-4a4e-a065-b74c1e8c83a7)

**Reply by Tyrrrz** [reply]
There is indeed a 1-second delay in the auto-generated subtitles for the video you linked

**Reply by lofti198** [reply]
Wow, I just got that he consider as well the timespan from previous word (I mean "system"), and in this case its really make sense. Iam no just thinking of how to better tackle this in my task of uniting subtitles to logical text blocks like sentences or paragraphs

**Reply by Tyrrrz** [reply]
Yes, each part's offset is relative to the caption's offset (to which the parts belong)
